packages/datatrak-web/src/features/Survey/Components/SurveySideMenu/SurveySideMenu.tsx

```diff
@@ -77,6 +88,14 @@ const SurveyMenuItem = styled(ListItem).attrs({
       overflow: hidden;
     }
   }
+
+  ${({ theme }) => theme.breakpoints.down('md')} {
+    border-bottom: 1px solid ${({ theme }) => theme.palette.divider};
+    ${({ $active }) => $active && `background-color: #f4f9ff;`}
```

Comment:
Mostly for devx, presuming you have the `styled-components` plugin (which I believe comes bundled with WebStorm)

(This suggestion will fail Prettier formatting)

```suggestion
    ${({ $active }) => $active && css`background-color: #f4f9ff;`}
```